src/components/book/Book.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../../redux/books/books';
+
+const Book = ({ id, title, catagory }) => {
```

Comment:
- [ ] Kindly remove the category from here as it is undefined and causing an error or you can give it a default value.